bcda/api/requests_test.go

```diff
@@ -219,7 +232,13 @@ func (s *RequestsTestSuite) TestRunoutDisabled() {
 // TestRequests verifies that we can initiate an export job for all resource types using all the different handlers
 func (s *RequestsTestSuite) TestRequests() {
 
-	h := newHandler([]string{"ExplanationOfBenefit", "Coverage", "Patient"}, "/v1/fhir", "v1", s.db)
+	resourceMap := map[string]DataType{
+		"Patient":              {Adjudicated: true},
+		"Coverage":             {Adjudicated: true},
+		"ExplanationOfBenefit": {Adjudicated: true},
+	}
+
+	h := newHandler(resourceMap, "/v1/fhir", "v1", s.db)
 
 	// Use a mock to ensure that this test does not generate artifacts in the queue for other tests
 	enqueuer := &queueing.MockEnqueuer{}
```

Comment:
Should we add a test for validating adjudicated? Please let me know if this is already done somewhere and I missed it.

Reply:
Do you mean a test for validating pre-adjudicated?  From the looks of it, the `TestRequests` function is validating v1 adjudicated requests.

Reply:
Is adjudicated always set to true, no matter the ACO model?

Reply:
No, we're having it be explicitly set, thus the need for having the ACO config models updated across the environments before this is merged.  If an ACO model does not have adjudicated listed in the configuration, they will get a permissions error when making a request for any of the CCW BFD data.

Reply:
So I believe all request tests are done using this dummy ACO: "ba21d24d-cd96-4d7d-a691-b0e8c88e67a5." Which has a cms_id of "A0002." Should we add another dummy model and test for a negative where adjudicated is not set?

Reply:
Ah ha, I see what you're saying now.  Yes I agree, we should ideally have tests for ACOs testing all 4 possibly states (or at least the first 3).
```
1. [adjudicated, pre-adjudicated]
2. [adjudicated]
3. [pre-adjudicated]
4. []
```